src/asm/gamepad.s

```diff
@@ -171,6 +171,43 @@ SoftReset:
 .assert * <= $cbd3
 .endif
 
+.ifdef _RANDOM_WILD_WARP_COUNT
```

Comment:
Can this "see" the value of `wildWarpCount`?  If so, you might be able to write `.if wildWarpCount < 16`?  I'm honestly not sure.  If not, this is fine as is.

Reply:
I gave it a shot, but it didn't work:
`Attempt 1 failed: Error: Expected a constant
  at gamepad.s:176:4
    at Preprocessor.evaluateConst (C:\Users\*******\Documents\GitHub\crystalis-randomizer\target\debug\bin\cryr:129973:11)
    at .if (C:\Users\*******\Documents\GitHub\crystalis-randomizer\target\debug\bin\cryr:129727:53)
    at Preprocessor.tryRunDirective (C:\Users\*******\Documents\GitHub\crystalis-randomizer\target\debug\bin\cryr:129965:5)
    at Preprocessor.pump (C:\Users\*******\Documents\GitHub\crystalis-randomizer\target\debug\bin\cryr:129759:21)
    at pump.next (<anonymous>)
    at Preprocessor.next (C:\Users\*******\Documents\GitHub\crystalis-randomizer\target\debug\bin\cryr:126984:43)
    at Assembler.tokens (C:\Users\*******\Documents\GitHub\crystalis-randomizer\target\debug\bin\cryr:128815:26)
    at asm (C:\Users\*******\Documents\GitHub\crystalis-randomizer\target\debug\bin\cryr:158303:10)
    at shuffleInternal (C:\Users\*******\Documents\GitHub\crystalis-randomizer\target\debug\bin\cryr:158335:33)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)`

Reply:
I was afraid that was the case - no worries, thanks for checking!